conf/modules.config

```diff
@@ -470,28 +470,15 @@ process {
 
 // VARIANT CALLING
 process{
-    // ALL
-    withName: 'CONCAT_.*' {
-        // For unknown reasons, CONCAT_VCF sometimes fails with SIGPIPE
-        // (exit code 141). Rerunning the process will usually work.
-        errorStrategy   = {task.exitStatus == 141 ? 'retry' : 'terminate'}
-        ext.args         = { params.no_intervals ?  "-n" : "" } //Why ConcatVCF is never run when no_intervals is set..
-    }
-    withName : 'BGZIP_VC_.*' {
-        publishDir       = [
-            enabled: false
-        ]
-    }
-
     // DEEPVARIANT
```

Comment:
since the BGZIP module is also not used anymore for the variantcallers, you can remove it from the modules.config too

Reply:
well spotted 👍

Reply:
On the same not the TABIX_VC processes are also no needed anymore